cloudinit/url_helper.py

```diff
@@ -310,7 +310,7 @@ def _cb(url):
 
 def wait_for_url(urls, max_wait=None, timeout=None,
                  status_cb=None, headers_cb=None, sleep_time=1,
-                 exception_cb=None, sleep_time_cb=None):
+                 exception_cb=None, sleep_time_cb=None, request_method=None):
```

Comment:
Probably should add a param docstring for request_method and should document the tuple that we now return